tkg/clusterclient/clusterclient.go

```diff
@@ -587,12 +587,21 @@ func (c *client) WaitForClusterInitialized(clusterName, namespace string) error
 		err = currentClusterInfo.RetrievalError
 
 		if err == nil {
-			// If cluster's ReadyCondition is False and severity is Error, it implies non-retriable error, so return error
+			// should retry in this condition
+			// For dimension env, sometimes it will take some time to connect to vcenter to create resources like cluster secret
```

Comment:
Can we rewrite this comment? I don't think it makes sense, or at least I don't understand it.
vCenter isn't Kubernetes, so we're not creating a cluster secret. And this doesn't feel like it would be limited only to Dell Dimension, so what is the real thing we're fixing here? I suspect the real answer is disk I/O causing etcd to be slower...


```suggestion
			// Retry waiting for cluster ready status for slow I/O infrastructure or resource constrained environments.
```